build.sc

```diff
@@ -5,24 +5,27 @@ import mill.scalajslib.ScalaJSModule
 import mill.scalalib._
 import mill.scalalib.api.CompilationResult
 
+val projectScalaVersion = "2.13.5"
+
 //noinspection ScalaFileName
+
```

Comment:
Any reason for blank line here? Some linter or IDE rules?

Reply:
That might have been an accident on my part